### ISSUE TITLE
Store Metadata to DocumentDB

### ISSUE THREAD

**Issue by noodlefrenzy** [noodlefrenzy/CognitiveServicesTutorial#10]
Open question - (where) do we store images? If not to DocDB, we should have a separate task for that and this should be updated to store the location of the stored image.

Store all metadata gathered from the cognitive services about the image, as well as filename, type, initial path, upload/processing date/time, time for each API call, total time to process.

**Comment by noodlefrenzy** [comment]
Code done in separate branch in .NET form, but waiting on potential of UWP code for accessing DocDB. @Foxman13 is pursuing REST API implementation.
